src/main/java/Main.java

```diff
@@ -0,0 +1,77 @@
+import feign.Feign;
+import feign.jackson.JacksonDecoder;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.openfeign.EnableFeignClients;
+import org.springframework.cloud.openfeign.support.SpringMvcContract;
+import uk.gov.hmcts.reform.fees.client.FeesApi;
+import uk.gov.hmcts.reform.fees.client.health.FeesHealthIndicator;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+
+@SpringBootApplication
+@EnableFeignClients
+public class Main {
```

Comment:
I am not quite convinced with this `Main` class in the code. I would rather prefer to have functional tests instead. Please refer to the example here : https://github.com/hmcts/ccd-client/tree/master/src/functionalTest/java/uk/gov/hmcts/reform/ccd/client